ui/plugins/controller.js

```diff
@@ -66,12 +66,12 @@ treeherder.controller('PluginCtrl', [
             var successTab = "jobDetails";
             var failTab = "failureSummary";
 
-            // show/hide Talos panel if this is a Talos job and default to
-            // Talos if the job was a ``success``
-            $scope.tabService.tabs.talos.enabled = (job.job_group_name.indexOf('Talos') !== -1);
-            if ($scope.tabService.tabs.talos.enabled) {
-                successTab = "talos";
```

Comment:
this patch removed successTab definition- so right now the default view will be jobDetails, in fact we would like it to be "successTab = "perfDetails".  Is there a specific reason to not do this?  I can imagine in the build jobs this would be confusing, but for all other test jobs it would make sense.